CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Change Log
-All notable changes to this project's source code will be documented in this file. Items under `Unreleased` is upcoming features that will be out in next version.
+All notable changes to this project's source code will be documented in this file. Items under `Unreleased` is upcoming features that will be out in next version. NOTE: major versions of the npm module and the gem must be kept in sync.
```

Comment:
@kaizencodes has a great point.  Since major-minor-patch-build/alpha/beta must match on both sides (?!), why not just direct-match the strings and be done with it?  What effect will this have on "local-development" activity against ReactOnRails directly ? @justin808 

If the desire is to match "only" M.M.P, then yes I agree rename the const VERSION_REGEX and also maybe pin the rx to begin-of-string `VERSION_REGEX = /^(\d+)\.(\d+)\.(\d+)/`